deploy/mariadb-password.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+data:
+  password: cGFzc3dvcmQ=
```

Comment:
How about we follow up with generating a random pw secret each time "make deploy" is run, instead of a static secret checked into git?